tests/e2e/api/src/repositories/rest/coupons/transformer.ts

```diff
@@ -57,6 +57,7 @@ export function createCouponTransformer(): ModelTransformer< Coupon > {
 					maximumAmount: 'maximum_amount',
 					emailRestrictions: 'email_restrictions',
 					usedBy: 'used_by',
+					links: '_links',
```

Comment:
Just curious: why the underscore here?

Reply:
That comes from the rest API. I suspect that it originated either from being a read-only field or a WordPress standard at some point in the past.